astropy/stats/bayesian_blocks.py

```diff
@@ -402,12 +394,32 @@ class Events(FitnessFunc):
     Parameters
     ----------
     p0 : float (optional)
-        False alarm probability, used to compute the prior on N
-        (see eq. 21 of Scargle 2012).  Default prior is for p0 = 0.
+        False alarm probability, used to compute the prior on N_blocks
+        (see eq. 21 of Scargle 2012).  Default prior is for p0 = 0.05.
+        For the Events type data, ``p0`` does not seem to be an accurate
+        representation of the actual false alarm probability. If you are using
+        this fitness function for a triggering type condition, it is
+        recommended that you run statistical trials on signal-free noise to
+        determine an appropriate value of ``gamma`` or ``ncp_prior`` to use
+        for a desired false alarm rate.
     gamma : float (optional)
         If specified, then use this gamma to compute the general prior form,
-        p ~ gamma^N.  If gamma is specified, p0 is ignored.
+        p ~ gamma^N_blocks.  If gamma is specified, p0 is ignored.
```

Comment:
Useful trick: if you do ``:math:`p \sim \gamma^{N_{\rm blocks}}` `` sphinx will render this as LaTeX in the HTML docs.  Important gotcha with that, though: you'll need to either double all backspaces, or put an ``r`` at the start of the docstring:

bad:
```
"""
:math:`\frac{1}{a}` Will fail, because it thinks \f is an escape character
"""
```
good:
```
r"""
:math:`\frac{1}{a}` Works because the "r" means "treat this as a raw string"
"""
```